docs/UserGuide.md

```diff
@@ -17,15 +17,15 @@ SocialBook is a **desktop app for managing contacts, optimized for use via a  Li
 
 1. Ensure you have Java `17` or above installed in your Computer.
 
-1. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103-F10-2/tp/releases).
+2. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103-F10-2/tp/releases).
```

Comment:
this is actually the convention for `.md` files. it will be rendered correctly in sequence and makes it easier to maintain, and less likely to make mistakes in numbering when inserting new items especially in between existing ones